source/common/grpc/transcoder_input_stream_impl.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "common/buffer/buffer_impl.h"
+
+#include "src/transcoder_input_stream.h"
```

Comment:
Can you wire up the external deps for this PR? Ideally, the grpc_transcoder library doesn't live under the `src` prefix, but something more like `grpc_transcoder/...`.